api/agent_test.go

```diff
@@ -1216,7 +1216,7 @@ func TestAPI_AgentMonitorJSON(t *testing.T) {
 		if err := json.Unmarshal([]byte(log), &output); err != nil {
 			t.Fatalf("log output was not JSON: %q", log)
 		}
-	case <-time.After(10 * time.Second):
+	case <-time.After(15 * time.Second):
```

Comment:
I don't think this should be in here.

Reply:
Yes, removed it since I fixed it in https://github.com/hashicorp/consul/pull/7541